Frontend/ui-common/src/FileProviders/AssetsProvider/AssetsProvider.js

```
@@ -20,6 +20,41 @@ export const getFromTopWindow = function (key) {
+  const mJOS = JSON.parse(JBridge.loadFileInDUI(fileName))
+  checkAndUpdateOverrides(mJOS);
+  return mJOS;
+}
+
+function checkAndUpdateOverrides(mJOS) {
+  const overrides = mJOS.overrides;
+  if (overrides != undefined && Array.isArray(overrides)) {
+    overrides.forEach(function (override) {
+      var shouldOverride = false;
+      try {
+        shouldOverride = eval(override.condition);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)